NCEP/cronjob_hera.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash --login
+
+echo "Job 1 is running"
+sh /scratch1/NCEPDEV/nems/AIML/graphcast/NCEP/gc_job1_hera.sh
```

Comment:
Can we call this: gc_prepdata_hera.sh to be more specific?

Reply:
updated